app/_locales/en/messages.json

```
@@ -3778,6 +3778,9 @@
   "replace": {
     "message": "replace"
   },
+  "reportProblem": {
+    "message": "Report an issue/problem"
```

```suggestion
    "message": "Report a problem"
```

no need to include `issue/` here